class/defaults.yml

```diff
@@ -113,6 +113,12 @@ parameters:
           secretNamespace: ${appcat:services:exoscale:secretNamespace}
           providerSecretNamespace: ${appcat:services:exoscale:providerSecretNamespace}
 
+      # Dev services for local development
+      dev:
+        # deploys a simple minio composition to provide local objectbuckets
+        # this composition implements the objectbucket xrd
+        enableLocalObjectBucket: false
+
```

Comment:
I'd rather not have any defaults related to local development. It invites people to try them on a cluster..

Reply:
second that